addons/udes_routing/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
We don't need to set this for Odoo 14, can we remove this from all Python files in the MR?